components/library/libjpeg6-ijg/libjpeg6.p5m

```diff
@@ -24,24 +24,10 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
 depend fmri=image/library/libjpeg6-ijg type=require
 
-# Documentation provided by default implementation
-file path=usr/share/man/man1/cjpeg.1
-file path=usr/share/man/man1/djpeg.1
-file path=usr/share/man/man1/jpegtran.1
-file path=usr/share/man/man1/rdjpgcom.1
-file path=usr/share/man/man1/wrjpgcom.1
+# Documentation and executables should default to libjpeg-turbo
+depend fmri=image/library/libjpeg-turbo type=require
```

Comment:
Why do we need this dependency?

Reply:
I did it the other way around I guess... I wanted that people who have libjpeg6 installed would get libjpeg-turbo installed automatically at the next update.

Reply:
I don't understand if I like it or not. If some package depends on libjpeg6-jig, it will have penalty of installing both. But we don't have such packages.
Feel free to do what you consider best.

Reply:
Most likely it does not make much difference since almost all of our packages have been migrated to libjpeg-turbo.